### PR TITLE
Add ability to choose a department from the API to livechat

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

Adds the ability to the website choose a deparment for that livechat session.

### CODE REVIEW

**Comment by engelgabriel** [comment]
@rodrigok can you please review this?

File: packages/rocketchat-livechat/app/client/lib/_livechat.js [hunk 1]
```diff
@@ -122,4 +127,11 @@ this.Livechat = new (class Livechat {
 	set room(roomId) {
 		this._room.set(roomId);
 	}
+	set department(departmentId) {
+		const dept = Department.findOne({ _id: departmentId }) || Department.findOne({ name: departmentId });
+
+		if (!dept) {
+			this._department.set(dept._id);
```
Comment (rodrigok) [inline packages/rocketchat-livechat/app/client/lib/_livechat.js]: Is this right? Shouldn't be `if (dept)`?

File: packages/rocketchat-livechat/app/client/lib/chatMessages.coffee [hunk 2]
```diff
@@ -76,20 +76,32 @@ class @ChatMessages
 			rid ?= visitor.getRoom(true)
 
 			sendMessage = (callback) ->
-				msgObject = { _id: Random.id(), rid: rid, msg: msg, token: visitor.getToken() }
+				msgObject = {
+					_id: Random.id(),
+					rid: rid,
+					msg: msg,
```
Comment (rodrigok) [inline packages/rocketchat-livechat/app/client/lib/chatMessages.coffee]: Just one tip, do you know we can use [property shorthand](http://es6-features.org/#PropertyShorthand)?
Example:
```javascript
msgObject = {
  _id: Random.id(),
  rid,
  msg,
  ...
}
```

Comment (sampaiodiego) [inline packages/rocketchat-livechat/app/client/lib/chatMessages.coffee]: can it be used on coffeescript already?

**Pull request merged**